src/corehost/cli/apphost/bundle/marker.cpp

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "marker.h"
+#include "pal.h"
+#include "trace.h"
+#include "utils.h"
+
+using namespace bundle;
+
+int64_t marker_t::header_offset()
+{
+    // Contains the bundle_placeholder default value at compile time.
+    // If this is a single-file bundle, the last 8 bytes are replaced 
```

Comment:
I believe it is the **first** 8 bytes that are replaced?

Reply:
Good catch - thanks a lot!
https://github.com/dotnet/runtime/pull/44874